src/spec/genre_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../Classes/genre'
+
+describe Genre do
```

Comment:
Please add a before each block after describe to initialize a music object and genre object for each test